src/integration-tests/permissions.js/permissions.spec.js

```diff
@@ -0,0 +1,48 @@
+const commonHelper = require('@commonTests')
+const schema = require('./responseSchema')
+
+describe('mentoring/v1/permissions ', function () {
+	let userDetails
+	beforeAll(async () => {
+		userDetails = await commonHelper.mentorLogIn()
+	})
+	it('/create', async () => {
+		let res = await request.post('/mentoring/v1/permissions/create').send({
+			code: 'edit_session',
+			module: 'session_edit',
+			actions: 'WRITE',
```

Comment:
@sumanvpacewisdom actions is array right ?

Reply:
yes , i did not change it in test case , will do it  @rakeshSgr